lib/amqp/session.rb

```diff
@@ -104,7 +105,7 @@ class Session < EM::Connection
     class << self
       # Settings
       def settings
-        @settings ||= AMQP::Settings.default
+        @settings ||= AMQ::Settings.default
```

Comment:
Do we need `AMQP.settings` method that fetches only `:logging` key? I checked the repo and didn't find any usage of it, but as far as it has comment about **global** behaviour, I thought one of underlying gems could rely on this parameter, from example `eventmachine`.